test/buffer/worker/utils_test.go

```diff
@@ -68,21 +67,21 @@ func (ts *testSuite) AssertEtcdState(expectedFile string) {
 // WaitForLogMessages wait until the lines are logged or a timeout occurs.
 // The lines do not have to be logged consecutively,
 // there can be another line between them, but the order must be preserved.
-func (ts *testSuite) WaitForLogMessages(timeout time.Duration, lines string) {
+func (ts *testSuite) WaitForLogMessages(timeout time.Duration, lines string) bool {
 	expected := `%A` + strings.ReplaceAll(strings.TrimSpace(lines), "\n", "\n%A") + `%A`
-	assert.Eventually(ts.t, func() bool {
-		return wildcards.Compare(expected, ts.logger.AllMessages()) == nil
-	}, timeout, 100*time.Millisecond, ts.logger.AllMessages())
+	return assert.Eventually(ts.t, func() bool {
+		return wildcards.Compare(expected, ts.logsOut.String()) == nil
+	}, timeout, 100*time.Millisecond, ts.logsOut.String())
 }
 
 func (ts *testSuite) AssertNoLoggedWarning() {
-	msgs := ts.logger.WarnMessages()
-	assert.Len(ts.t, msgs, 0, "Found some warning messages: %v", msgs)
+	msgs := ts.logsOut.String()
+	assert.Falsef(ts.t, strings.Contains(msgs, "WARN"), "Found some warning messages: %v", msgs)
```

Comment:
Drobne zmeny v tychto assertoch, predtym sa pouzival `DebugLogger`, ... teraz je to iba jednoduchy text stream stdout/stderr z jednotlivych nodes (do AtomicWriter).